README.md

```diff
@@ -53,6 +53,8 @@ Execute the following command to start the docker containers:
 ```
 docker compose up -d
 ```
+Note: In Docker versions before 19.03, use `docker-compose up -d` as the command syntax.
```

Comment:
this might not work with current `docker-compose.yaml` https://github.com/Layr-Labs/eigenda-operator-setup/blob/master/docker-compose.yml#L31 - I think `readonly` needs to change to `ro` as I think `readonly` is not backward compatible. Please check this assumption.